examples/rss-feed/Readme.md

```diff
@@ -0,0 +1,179 @@
+
+# RSS Feed Example
+
+This example uses Metalsmith to create a static site and implements an rss feed of all your posts using the metalsmith-feed and metalsmith-collections plugins. 
+
+## Install Node and Update NPM 
```

Comment:
For running locally I'd recommend using nvm/nvm-windows to install node - `nvm install node` will install the latest stable version.